usr/lib/common/aix/secure_getenv.h

```diff
@@ -0,0 +1,13 @@
+#ifndef SECURE_GETENV_H
```

Comment:
Where does this implementation come from? If you wrote it yourself, then this file needs a copyright and license header, otherwise we should include the original header....

Reply:
Yes, please add the 'normal' copyright/license header as used by other opencryptoki source files.